### ISSUE TITLE
[Bug] Accessibility : Vocalise the icon added to the list item (in Lists) 

### ISSUE THREAD

**Issue by afrahB** [Orange-OpenSource/ods-android#337]
### Prerequisites

- [X] I have [searched the backlog](https://github.com/Orange-OpenSource/ods-ios/issues) for duplicate or closed feature requests

### Your test device

Samsung S20

### OS Version

Android 10

### App version

1.0

### Describe the issue

Go to components -> Lists -> Lists demon 

- Click "Icon" in Trailing 
- Activate Talkback
- browse the list for vocalisation
- no vocalisation for the icon (here it s a clickable icon so it should be vocalised)

### Expected Behavior

Vocalise Icons in list items.
